<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActuatorDialog</class>
 <widget class="QDialog" name="ActuatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>900</number>
   </property>
   <property name="maximum">
    <number>2100</number>
   </property>
   <property name="value">
    <number>1500</number>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_setpoint">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>95</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Setpoint</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckb_tmp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Request Temperature</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckb_pos">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Request Position</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckb_cir">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Request Circuit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckb_act">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Actuate Servo</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>491</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_0">
    <attribute name="title">
     <string>Step</string>
    </attribute>
    <widget class="QPushButton" name="btn_tab0_left">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>90</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>900
A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_tab0_right">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2100
D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_tab0_mid">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1500
S</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Ramp</string>
    </attribute>
    <widget class="QLineEdit" name="le_velocity">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_velocity">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>109</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>us/sec</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_tab1_left">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>90</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Left
A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_tab1_mid">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Central
S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_tab1_right">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Right
D</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Manual</string>
    </attribute>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimum">
      <number>900</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
     <property name="value">
      <number>1500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
